source/CoordinateConversion/CoordinateConversionLibrary/Helpers/ExportCSV.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Copyright 2016 Esri 
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Source: https://github.com/jitbit/CsvExport
```

Comment:
Since this is third party code - the above header/copyright should be removed. Also the license says it is MIT, so I would probably just say that and link to the repo/license to credit/license: https://github.com/jitbit/CsvExport#license  

Also consider adding a "Credits" section to the readme ex: https://github.com/Esri/military-symbol-editor-addin-wpf/#credits that credits this code